sbom.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://cyclonedx.org/schema/bom-1.5.schema.json",
+    "bomFormat": "CycloneDX",
+    "specVersion": "1.5",
+    "serialNumber": "urn:uuid:dc42a43b-4ace-4c42-9a6e-0b9e28fdd100",
+    "version": 1,
+    "metadata": {
+        "timestamp": "2024-05-08T09:51:01Z",
+        "tools": [
+            {
+                "name": "composer",
+                "version": "2.7.6"
+            },
+            {
+                "vendor": "cyclonedx",
+                "name": "cyclonedx-php-composer",
```

Comment:
This file looks generated. Where did you get it from?

Reply:
The file was generated using the cyclonedx/cyclonedx-php-composer library to generate an SBOM from composer.lock, then manually removing all components and dependencies as we only report bundled dependencies:
```shell
composer require --dev cyclonedx/cyclonedx-php-composer
composer CycloneDX:make-sbom --output-file=sbom.json --output-format=JSON --omit=dev --omit=plugin
```